inde.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+​
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>ToDoリスト</h1>
+    ​
+    <div>
+      <input id="allbtn" type="radio" name="level" value="ok" onchange="changeBtn()"　 checked />
```

Comment:
全角の空白が入ってしまっていて、正常に動かなくなっています。
```suggestion
      <input id="allbtn" type="radio" name="level" value="ok" onchange="changeBtn()" checked />
```